src/js/WebParts/DynamicPortfolio/index.tsx

```diff
@@ -642,13 +642,8 @@ export default class DynamicPortfolio extends BaseWebPart<IDynamicPortfolioProps
                 updatedState.groupBy = groupByColumn;
             }
         }
-        await this.updateState(updatedState);
 
-        // Check if the searchbar had searchterm on new view
-        if (updatedState.searchTerm) {
-            document.querySelectorAll('input[id^="SearchBox"].ms-SearchBox-field')[0].attributes[4].value = this.state.searchTerm;
-            // TODO: Make the search value stay when use clicks the search bar after view change, right now it dissapears
-        }
+        await this.updateState(updatedState);
```

Comment:
Putting the await below makes it break. It should be as before as the updateState clears the searchform.

Reply:
Have you tested after my commit?

Reply:
(Shouldn't use document.querySelectorAll code like this in a React app.)

Reply:
> Have you tested after my commit?

Didn't see the commits you made above, I've tested it and it works. I was certain that I had already tried this and therefore ended up with the `document.querySelectorAll` method of doing it.